fix: chrome's memory saver seems to cause spurious preview disconnects (#209)
stackblitz/docs: docs/platform/webcontainers/browser-config.md

In some browsers, this feature is blocked by “third-party cookie” or “third-party storage” restrictions. These are legitimate restrictions when the third-party domain is an ad server or a tracking server, but in the case of WebContainers the third-party domain is where your project code runs.

## Chrome

> With chrome's defaults, starting with version 118 or later, you should get a pretty good experience out of the box.

Reviewer: ```suggestion
With Chrome's defaults, starting with version 118 or later, you should get a pretty good experience out of the box.
```

> If you run into issues or want to have an even smoother experience, read-on.

Reviewer: ```suggestion
Read on if you run into issues or want an even smoother experience.
```

> Previews that are opened in a separate tab will require you that you connect them to the editor. You usually get prompted to click on a button to do that. However with Chrome [memory saver][CHROME_MEMORY_SAVER], you might find yourself doing that more frequently than expected. To avoid it, you can add the following exception in [chrome://settings/performance](chrome://settings/performance):

Reviewer: ```suggestion
Previews opened in a separate tab will require you to connect them to the editor. You usually get prompted to click on a button to do that. However, you might do that more frequently than expected with Chrome's [memory saver][CHROME_MEMORY_SAVER]. To avoid it, you can add the following exception in [chrome://settings/performance](chrome://settings/performance):
```

```
https://stackblitz.com
```

> Once done you should see something similar to this:

Reviewer: ```suggestion
Once done, you should see something similar to this:
```
